packages/cardano-services-client/src/HandleProvider/KoraLabsHandleProvider.ts

```diff
@@ -44,10 +44,12 @@ export const toHandleResolution = ({
   apiResponse: IHandle;
   policyId: Cardano.PolicyId;
 }): HandleResolution => ({
+  backgroundImage: Asset.Uri(`ipfs://${apiResponse.background}`),
```

Comment:
I have noticed that the interface added for HandleResolution here, at the top of the file matches the HandleResolution from the HandleProvider, the native one... but not the KoralabsType in the `core/provider/handleProvider/types.ts` .. so I am unsure this works... I think that they do return images though, but the interface may need to change the type here in that package, and then we can map the properties to return the same names as on the other provider if we want.. 
[KoraLabs](https://github.com/koralabs/handles-public-api-interfaces/blob/master/index.ts#LL94C1-L94C1) interface